lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1054,10 +1069,34 @@ def control_state(vm, vms_service, module):
             )
 
 
+def wait_for_import(module, vms_module, vms_service):
+    if module.params['wait']:
+        start = time.time()
+        timeout = module.params['timeout']
+        poll_interval = module.params['poll_interval']
+        while time.time() < start + timeout:
+            vm = _fetch_vm(module, vms_module, vms_service)
+            if vm:
+                return vm
+            time.sleep(poll_interval)
+
+
+def _fetch_vm(module, vms_module, vms_service):
+    if (module.params['id'] is not None):
```

Comment:
Same as in `ovirt_templates` patch, can you reimplent the support in `ovirt.py` ?